bin/check_formatting.dart

```diff
@@ -0,0 +1,17 @@
+import 'dart:async';
+import 'utils.dart';
+
+Future<Null> main() async {
+  final PresubmitUtils utils = new PresubmitUtils();
+
+  await utils.fetchConfiglet();
+
+  print('Checking config.json formatting...');
+  await utils.run('bin/configlet', ['lint', '.']);
```

Comment:
How do we know that the `fetchConfiglet` was a success? Shouldn't there be a check before executing `configlet fmt .` ?